src/app/pages/start/containers/start-page.component.ts

```diff
@@ -1,52 +1,49 @@
 import 'rxjs/add/operator/delay'
 
 import { Component } from '@angular/core'
-import { LoadingController, NavController, NavParams } from 'ionic-angular'
+import { NavController, NavParams } from 'ionic-angular'
 
-import { Assessment } from '../../models/assessment'
-import { Question } from '../../models/question'
-import { Task } from '../../models/task'
-import { AnswerService } from '../../providers/answer-service'
-import { TimeStampService } from '../../providers/timestamp-service'
-import { QuestionsPage } from '../questions/questions'
+import { Assessment } from '../../../shared/models/assessment'
+import { Question } from '../../../shared/models/question'
+import { Task } from '../../../shared/models/task'
+import { QuestionsPageComponent } from '../../questions/containers/questions-page.component'
+import { StartService } from '../services/start.service'
 
 @Component({
   selector: 'page-start',
-  templateUrl: 'start.html'
+  templateUrl: 'start-page.component.html'
 })
-export class StartPage {
+export class StartPageComponent {
   associatedTask: Task
   introduction: String = ''
   title: String = ''
   questions: Question[]
   endText: String
+  assessment: Assessment
 
   constructor(
     public navCtrl: NavController,
     public navParams: NavParams,
-    private answerService: AnswerService,
-    private timeStampService: TimeStampService
+    private startService: StartService
   ) {
     this.associatedTask = this.navParams.data.associatedTask
     this.title = this.navParams.data.title
     this.introduction = this.navParams.data.introduction
     this.questions = this.navParams.data.questions
     this.endText = this.navParams.data.endText
+    this.assessment = this.navParams.data.assessment
   }
 
-  ionViewDidLoad() {}
-
   ionViewDidEnter() {
-    this.answerService.reset()
-    this.timeStampService.reset()
```

Comment:
Maybe the answer service needs to be reset here otherwise lingering answers from pervious questionnaire could get merged with this questionnaire (or maybe if reset before navigating to this page then not required) ?